ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -324,6 +324,34 @@ spec:
         - name: dest-registry-credentials
           workspace: registry-push-credentials
 
+    - name: set-cert-project-repository
+      runAfter:
+        - copy-bundle-image-to-released-registry
```

Comment:
The task `create-container-image` depends on `copy-bundle-image-to-released-registry`. You should probably change a dependency graph since you are adding a new step here.

Reply:
Fixed